app/_locales/en/messages.json

```diff
@@ -1194,6 +1200,9 @@
   "settings": {
     "message": "Settings"
   },
+  "safelloBuy": {
+    "message": "Buy with Safello"
```

Comment:
Do we still need this message? It seems... overly burdensome on the translation team to ask for "Buy with Safello" _and_ "Buy ETH with Safello" when we could just use the former in both places.